### PR TITLE
Some additional i18n explanation

### PR DESCRIPTION
**Pull request by benjaoming**
For https://github.com/learningequality/kolibri/pull/4368

### CODE REVIEW

File: docs/references/release_process.rst [hunk 1]
```diff
@@ -31,13 +31,15 @@ String freeze and translation
 
 Once we are close to stabilizing the UI, we should schedule a string freeze. This date is the time after which no user-facing text in the application can be changed, because after string freeze we will begin getting quotes from translators and initiating the translation process.
 
-On the string freeze date, the strings for the upcoming release should be uploaded to crowdin as described in :ref:`crowdin`. Use the stats outputted during that process to get quotes from our translators, and choose start and end dates for translations to occur in.
+You will need to log in to CrowdIn to create a new Version Branch, following the existing naming scheme ``release-vN.N.x``. CrowdIn will automatically reuse previously translated strings.
```
Comment (indirectlylit) [inline docs/references/release_process.rst]: This isn't quite accurate - the new script will automatically create the branch for you:

https://kolibri-dev.readthedocs.io/en/develop/references/i18n.html#exporting-and-uploading

File: docs/references/release_process.rst [hunk 2]
```diff
@@ -31,13 +31,15 @@ String freeze and translation
...
+
+Strings from the Perseus plugin should also be revisited. They are maintained in the repo `learningequality/kolibri-exercise-perseus-plugin <https://github.com/learningequality/kolibri-exercise-perseus-plugin>`__, but the source strings are generated and uploaded through the main ``kolibri`` project. The translated files from CrowdIn such as ``exercise_perseus_render_module-messages.json`` are downloaded through the ``kolibri-exercise-perseus-plugin`` project.
 
 Before the translation start date, provide time to do one final review of all user-facing strings. This can be done e.g. in the course of doing a preliminary translation into Spanish.
 
-When the user-facing strings have been signed off, notify translators that translation can begin.
+When the user-facing strings have been signed off, notify the i18n team that translation can begin.
```
Comment (indirectlylit) [inline docs/references/release_process.rst]: The i18n team should already be aware.. what I was trying to say was that the i18n team should notify our external translators that we're ready to begin

File: docs/references/release_process.rst [hunk 3]
```diff
@@ -31,13 +31,15 @@ String freeze and translation
...
+You will need to log in to CrowdIn to create a new Version Branch, following the existing naming scheme ``release-vN.N.x``. CrowdIn will automatically reuse previously translated strings.
 
-Remember to also include strings from the Perseus plugin if necessary.
+On the string freeze date, the strings for the upcoming release should be uploaded to crowdin as described in :ref:`crowdin`. You can run ``make i18n-crowdin-stats branch=<release-vN.N.x>`` to get figures on the translation work. Send the stats to the i18n team, whom can use them for communication with translators to measure the translation efforts needed.
+
+Strings from the Perseus plugin should also be revisited. They are maintained in the repo `learningequality/kolibri-exercise-perseus-plugin <https://github.com/learningequality/kolibri-exercise-perseus-plugin>`__, but the source strings are generated and uploaded through the main ``kolibri`` project. The translated files from CrowdIn such as ``exercise_perseus_render_module-messages.json`` are downloaded through the ``kolibri-exercise-perseus-plugin`` project.
```
Comment (indirectlylit) [inline docs/references/release_process.rst]: the process for perseus has been slightly updated:

https://kolibri-dev.readthedocs.io/en/develop/references/i18n.html#updating-the-perseus-plugin

**Review by indirectlylit — COMMENTED** [review]
thanks for the suggestions! A couple comments .. maybe you can help me get it into the final form

File: docs/references/release_process.rst (hunk 2 above)
Comment (benjaoming) [reply docs/references/release_process.rst]: Who is the reader of this doc?

File: docs/references/release_process.rst (hunk 1 above)
Comment (benjaoming) [reply docs/references/release_process.rst]: Oh okay, how nice :)

File: docs/references/release_process.rst (hunk 3 above)
Comment (benjaoming) [reply docs/references/release_process.rst]: Amending now...

Comment (benjaoming) [reply docs/references/release_process.rst]: Should we delete the CrowdIn interaction from the perseus project and just reference installing it in editable mode in Kolibri?

**Comment by benjaoming** [comment]
@indirectlylit I think I've resolved the feedback, have a look when you have time :)

File: docs/references/release_process.rst (hunk 2 above)
Comment (indirectlylit) [reply docs/references/release_process.rst]: I was writing for anyone on the LE team with a step to check off on the release checklist

File: docs/references/release_process.rst (hunk 3 above)
Comment (indirectlylit) [reply docs/references/release_process.rst]: good point the crowdin docs on Perseus are now out of date.

I'll submit a PR there soon... if you wouldn't mind opening an issue I'd appreciate!

**Review by indirectlylit — APPROVED** [review]
thanks for the tweaks

**Pull request merged**